src/core/hle/service/nvdrv/devices/nvhost_as_gpu.cpp

```diff
@@ -1,21 +1,31 @@
-// SPDX-FileCopyrightText: Copyright 2018 yuzu Emulator Project
-// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-FileCopyrightText: 2021 yuzu emulator team, Skyline Team and Contributors
+// (https://github.com/skyline-emu/)
+// SPDX-License-Identifier: GPL-3.0-or-later Licensed under GPLv3
+// or any later version Refer to the license.txt file included.
```

Comment:
This has been relicensed from GPL-2.0-or-later to GPL-3.0-or-later. Is this intentional/required? (note that new contributions can be licensed under the GPL-2.0-or-later without issues, as yuzu as a whole can be distributed under the GPL-3.0-or-later and use Apache-2.0 files - I'm not a lawyer this is not legal advice etc etc)

```suggestion
// SPDX-FileCopyrightText: 2021 yuzu Emulator Project
// SPDX-FileCopyrightText: 2021 Skyline Team and Contributors <https://github.com/skyline-emu>
// SPDX-License-Identifier: GPL-3.0-or-later
```